types/underscore/index.d.ts

```diff
@@ -5692,7 +5692,7 @@ declare module _ {
         * Wrapped type `object`.
         * @see _.values
         **/
-        values(): _Chain<T>;
+        values(): _Chain<any>;
```

Comment:
Why the weakening from `T` to `any` here? Was `T` the wrong type?

Reply:
Also; this is what `values(object: any)` returns (the variant without `Chain`).